devicetypes/vision-raytseng/vision-4-in-1-motion-sensor.src/vision-4-in-1-motion-sensor.groovy

```diff
@@ -0,0 +1,552 @@
+/**
+ *  Vision 4-in-1 Motion Sensor
+ *
+ *  Author: Ray Tseng
+ */
+metadata {
+  definition (name: "Vision 4-in-1 Motion Sensor", namespace: "vision-raytseng", author: "Ray Tseng", vid: "generic-motion-8", ocfDeviceType: "x.com.st.d.sensor.motion") {
+    capability "Battery"
+    capability "Motion Sensor"
+    capability "Relative Humidity Measurement"
+    capability "Temperature Measurement"
+    capability "Illuminance Measurement"
+    capability "Tamper Alert"
+    capability "Health Check"
+    
+    fingerprint mfr:"0109", prod:"2021", model:"2112", deviceJoinName: "Vision ZP3113-7 4-in-1 Motion"
+  }
+
+  tiles(scale: 2) {
```

Comment:
Please remove the whole tiles section since tiles are not supported in the SmartThings app.